### ISSUE TITLE
[Vanha] Add tumblestones to the drops of Pokemon that have spawners in Vanha

### ISSUE THREAD

**Issue by Shika-The-Deer** [NebulaMC-GG/Suggestions#162]
The Hisuian Pokeballs are a cool set of balls, but obtaining the tumblestones to make any decent amount of them is difficult when there's not a reliable way to farm them and the higher rarities give you a 3->1 ratio of stones to balls. This holds especially true for Sky Tumblestones, which can only drop from 5 Pokemon total, with the Pokemon they drop from being far too rare to viably farm. Even with the other two variations being less rare than the Sky Tumblestones, I feel it would still be worth adding them to the drops of Pokemon that have spawners in Vanha.

I don't believe that this would be overpowered compared to farming the normal Pokeballs because the highest catch rate of the bunch is 2.5x (Gigaton Balls), and Apricorn farms allow people to make a bunch of the stronger balls like Dusk and Quick Balls without having to farm as many Pokemon drops, needing only to get the metals from Metang and Bronzor spawners.

If there are already spawners that drop these items that I didn't know about feel free to poke fun at me kthxbye. <3

**Comment by Chunkkss** [comment]
Tumblestone grows, it's a farmable material. 

Instead of changing drops, we need to make tumblestone able to be placed like you can in creative. 
Then players will be able to farm both Tumblestone, and Apricorns.

**Comment by Khrisstias** [comment]
> Tumblestone grows, it's a farmable material.
> 
> Instead of changing drops, we need to make tumblestone able to be placed like you can in creative. Then players will be able to farm both Tumblestone, and Apricorns.

Yes you can farm it, but it takes a while to grow and a long time to break. you also need to use 3 per lid which is absurd for just farming them.
